src/pages/Doyun/Login/Login.scss

```
@@ -0,0 +1,49 @@
+.login {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  height: 500px;
+  width: 400px;
+  margin: 25px auto;
+  border: 1px solid #bbbbbb;
+  background-color: #ffffff;
+  .mainLogo {
+    font-family: 'Lobster', cursive;
+    font-weight: 50;
+    font-size: 2.5rem;
+    margin-bottom: 40px;
+  }
+  input {
+    background-color: #fafafa;
+    border: 1px solid #ebebeb;
+    margin-bottom: 15px;
+    padding: 10px 5px;
+    width: 250px;
+    .idInput {
+      font-family: 'Quicksand', sans-serif;
+    }
+    .passwordInput {
+      font-family: 'Quicksand', sans-serif;
+    }
+  }
```

```suggestion
    width: 250px;
    
    .idInput, .passwordInput {
      font-family: 'Quicksand', sans-serif;
    }
  }
```
- 동일한 스타일이니까 모아서 스타일링 해주시면 될거 같아요
- 그런데 앱 전체에서 다른 폰트가 기본적으로 적용되다가 폰트가 여기서만 바뀌나요?

---

앗 .. 확인 결과 불필요한 폰트였네요. 삭제 완료했습니다😅